webplugin/js/app/zendesk-chat-service.js

```diff
@@ -83,6 +86,15 @@ function ZendeskChatService() {
             });
         }
     };
+    _this.updateNumberInZopim = function() {
+        if(zChat.getVisitorInfo().phone != phoneNumber){
+            zChat.setVisitorInfo({ phone: phoneNumber }, function(err) {
```

Comment:
this will get called even if the phone number is not provided?

Reply:
so now we will keep calling Zendesk APIs getVisitorInfo even if we have nothing to update

Reply:
will place an if condition above this to check if phoneNumber exists or not

Reply:
only then it'll call getVisitorInfo

Reply:
read this https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND